CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-09-24
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+
+### Release notes 
+* Behavioral changes
+  * If your VM SKU does not support ephemeral or PremiumSSD OS disks, AKS will now use StandardSSD as the default OS disk type. Before, only VM SKUs that did not support ephemeral OS disks had this default.
```

Comment:
>  AKS will now use StandardSSD as the default OS disk type

as opposed to what currently?

Reply:
Hmm, per the sentence here isn't it simpler to say, if your sku doesn't support ephemeral or premium it will now use standard ssd, before it would use HDD.